src/main/java/com/quizapp/user/UserCreationDto.java

```diff
@@ -0,0 +1,15 @@
+package com.quizapp.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+
+public class UserCreationDto {
+
+    private String username;
```

Comment:
Do we have a username? Maybe it's identifier?

In addition, keeping id as the first variable in class is a good practice.